src/test/java/com/iexec/core/sms/SmsServiceTests.java

```diff
@@ -56,4 +59,15 @@ void shouldGetEmptyAddressForStandardTask() {
                 .isEqualTo(BytesUtils.EMPTY_ADDRESS);
         verify(smsClient, never()).generateTeeChallenge(anyString());
     }
+
+    @Test
+    void shouldGetEmptyAddressOnFeignException() {
```

Comment:
```suggestion
    void shouldGetEmptyAddressOnFeignException() {
```
```suggestion
    void shouldNotGetEnclaveChallengeOnFeignException() {
```

(to avoid confusion with `.isEqualTo(BytesUtils.EMPTY_ADDRESS)`)